docs/configuration.asciidoc

```diff
@@ -206,20 +207,20 @@ This table outlines the attributes and their corresponding permissions:
 |Attribute |Used in |Requires permission
 
 |`net.host.connection.subtype`
-|HTTP Spans
+|All Spans
 |https://developer.android.com/reference/android/Manifest.permission#READ_PHONE_STATE[READ_PHONE_STATE]
 
 |===
 
 [float]
 ==== Session ID configuration
 
-The Elastic agent provides a "session ID" for every span it creates.
+The Elastic agent provides a "session ID" for every span and log event it creates.
 The idea of a session is to provide a context that covers many transactions that a user did in order to fulfil their needs using an application.
-For most apps, a session could start when the user opens the app, and end when the user closes the app, or when the app is forced to get closed due to an unexpected error.
+For most apps, a session could start when the user opens the app, and end when the user closes the app.
 But for other apps, such as a ticketing app for a queue in a bank for example, the app will always be open, but a session might start when a person starts the process to get a new ticket, and end when the ticket is printed.
 
-By default, the Elastic agent generates a session ID on every app launch and keeps it until the app is terminated, or generates a new one after an idle period of time of 30 minutes.
+By default, the Elastic agent generates a session ID on app launch and keeps it until after an idle period of time of 30 minutes, restarting the timeout time on every APM-related transaction.
```

Comment:
This sentence was tough for me to digest. What about something like this?

```suggestion
By default, the APM agent generates a session ID on app launch. Each session is kept active until 30 minutes of idle time has elapsed. The idle time clock resets on every APM-related transaction.
```

Reply:
Love it, thanks!